contracts/types.md

```diff
@@ -1,32 +1,35 @@
-Types is a library contract of protocol structs. [View the code on GitHub](https://github.com/airswap/airswap-protocols/tree/master/protocols/swap).
+Types is a library contract of protocol structs. [View the code on GitHub](https://github.com/airswap/airswap-protocols/tree/master/source/types).
```

Comment:
` of protocol structs` -> also has functions and constants for the Swap contract